### PR TITLE
firefox: show popup when click "vault is locked"

### PR DESCRIPTION
**Pull request by lazyc97**
#272

officially supported by only firefox so far: [openPopup](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/browserAction/openPopup).

This function also exist on Chromium as I tested but it is not in the docs so I guess I shouldn't include it.

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -484,7 +487,17 @@ var bg_isBackground = true,
             setSidebarActionText(theText, tabId);
         }, function () {
             if (contextMenuEnabled) {
-                loadNoLoginsContextMenuOptions(bg_i18nService.vaultLocked);
+                if (bg_utilsService.isFirefox() && chrome.browserAction.openPopup) {
```
Comment (kspearrin) [inline src/background.js]: I would think it would be fine to just check `chrome.browserAction.openPopup`?

Comment (lazyc97) [reply src/background.js]: `chrome.browserAction.openPopup` exist in chrome but require a callback function while firefox require no arguments. There is a [discussion here](https://bugs.chromium.org/p/chromium/issues/detail?id=436489), this feature is not official in chrome so I'm not sure if it is safe to use.

**Comment by lazyc97** [comment]
Moved.

**Comment by kspearrin** [comment]
Thanks!

**Pull request merged**